### PR TITLE
Oemc 46 de create modals for all sharing options

### PR DESCRIPTION
**Pull request by mluena**
### Overview

_This PR adds the sharing options panel in map controls._
Screenshot 2023-09-14 at 06 55 31

- Changes icons for react-icons
- Removes unnecessary exports

*Note: there are some comments pending on designs review

### Designs

_[Link to the related design prototypes (if applicable)](https://www.figma.com/file/PyAxTEOtMjmW558rdmcwcy/OpenEarthMonitor-%5BInternal%5D?node-id=926%3A5005&mode=dev)_

### Testing instructions

_Map section -> map controls -> Share._

### Feature relevant tickets

_[Link to the related task manager tickets - share](https://vizzuality.atlassian.net/browse/OEMC-54?atlOrigin=eyJpIjoiYTg2YjkzMWM2OGMwNDIxYjgzYWZkOGM1MjdmMDFjNTAiLCJwIjoiaiJ9)_
_[Link to the related task manager tickets - icons](https://vizzuality.atlassian.net/browse/OEMC-70?atlOrigin=eyJpIjoiMTlhOWRkYWUzYTUxNGFmMjljNzYyYmMwMDAwZmRmZDciLCJwIjoiaiJ9)_
---

## Checklist before submitting

- [ ] Meaningful commits and code rebased on `develop`.
- [ ] Update CHANGELOG
- [ ] If this PR adds feature that should be tested for regressions when
      deploying to staging/production, please add brief testing instructions
      to the deploy checklist

### CODE REVIEW

File: src/components/map/controls/share/index.tsx [hunk 1]
```diff
@@ -0,0 +1,117 @@
+import { FC, useState, MouseEvent } from 'react';
+
+import { CopyToClipboard } from 'react-copy-to-clipboard';
```
Comment (davidsingal) [inline src/components/map/controls/share/index.tsx:3]: Before adding this lib, did you consider if you can do it yourself? I think is small enough to replicate what you need in your code.
https://github.com/nkbt/react-copy-to-clipboard/blob/master/src/Component.js

File: src/components/datasets-list/datasets-item-header/index.tsx [hunk 2]
```diff
@@ -2,37 +2,42 @@
...
-        {downloadUrlBase && (
-          <a
-            className="cursor"
-            href={`${downloadUrlBase}`}
-            target="_blank"
-            rel="noopener noreferrer"
-          >
-            <Icon icon={DOWNLOAD_SVG} className="cursor h-6 w-6 text-gray-300" />
+      <div className="flex items-baseline space-x-2 pt-2">
+        {<Info layer_id={layer_id} />}
```
Comment (davidsingal) [inline src/components/datasets-list/datasets-item-header/index.tsx:33]:
```suggestion
        <Info layer_id={layer_id} />
```

File: src/components/datasets-list/datasets-item-info/index.tsx [hunk 3]
```diff
@@ -1,10 +1,10 @@
 'use client';
 
+import { LuInfo } from 'react-icons/lu';
```
Comment (davidsingal) [inline src/components/datasets-list/datasets-item-info/index.tsx:3]: Unify all libraries, it should be from heroicons. Remove `react-icons`

**Review by davidsingal — CHANGES_REQUESTED** [review]

File: src/components/datasets-list/datasets-item-info/index.tsx (hunk 3 above)
Comment (mluena) [reply src/components/datasets-list/datasets-item-info/index.tsx:3]: I used heroicons and after agreed with Ari to use react-icons so I changed everything

**Pull request merged**